changelog/fix_github_actions_in_non_default_directory.md

```diff
@@ -0,0 +1 @@
+* [#9933](https://github.com/rubocop/rubocop/pull/) Fix GitHub Actions formatter when running in non-default directory ([@ojab][])
```

Comment:
`% bundle exec rspec spec/project_spec.rb` has failure. Can you update to the below?

```diff
-* [#9933](https://github.com/rubocop/rubocop/pull/) Fix GitHub Actions formatter when running in non-default directory ([@ojab][])
+* [#9933](https://github.com/rubocop/rubocop/pull/9933): Fix GitHub Actions formatter when running in non-default directory. ([@ojab][])
```

Reply:
:facepalm: pushed.